src/RepoAuditor/Display.py

```diff
@@ -37,16 +37,16 @@ def GetInternalPanelContent(
     if module.description:
         internal_content.append(f"{module.description.strip()}\n")
 
-    internal_content.append(
-        textwrap.dedent(
-            f"""\
-                Successful:     {num_success} ({(num_success / num_requirements):.02%})
-                Warnings:       {num_warning} ({(num_warning / num_requirements):.02%})
-                Errors:         {num_error} ({(num_error / num_requirements):.02%})
-                Skipped:        {num_does_not_apply} ({(num_does_not_apply / num_requirements):.02%})
-                """,
-        ),
+    metrics_display = textwrap.dedent(
```

Comment:
Nit: The indentation seems to be off here (too much indentation).

Reply:
Yup, I just noticed that myself. Fixing it. :)